Product proposal template.md

```diff
@@ -1,30 +1,33 @@
 ## High Level Product Description Template:
 
-In addition to the below, proposed KBase apps should also follow the requirements in the [KBase product requirements document](./KBase%20product%20requirements.md)
+*Proposed KBase apps should also follow the requirements in the [KBase product requirements document](./KBase%20product%20requirements.md). Products that are not apps should follow those requirements where appropriate.*
 
-These high level descriptions must include the following:
+For most types of products, product descriptions should include the following:
 
 ### Summary
-A brief summary of what the product will enable or improve for the user.
+A brief summary of what the product will enable or improve for the user. (In other words, WHY should we develop this product?)
+
+### Keyword(s)
+What areas of science or technology does this product support?
 
 ### Story/Description
-A longer story about the what the functionality is designed to do; what its significance and impact would be; and an argument for why now?
+A longer story describing what the functionality is designed to do; what its significance and impact would be; and an argument for why now?
 
 ### User stories
 A small set of user stories that discuss how this functionality would improve the system for the users.
 
 ### Narrative Mockup
-A mocked-up Narrative (written in Markdown) that:
+Where appropriate, a mocked-up Narrative (written partly or entirely in Markdown) that:
   1. Shows a plausible, scientifically coherent, story of how the new functionality would appear to the user--essentially, an annotated example of how a user would experience the new behaviors. This could be new scientific functionality, or simple better experience.
   2. Clear incremental, demonstrable deliverables on the road to the full functionality. (If possible, deliverables should be scoped to be demonstrable after each two-week sprint.)
   3. If the new functionality requires management or analysis of data, then exemplar data sets and their logic for inclusion and testing must be made available and uploaded if possible into the mock-narrative.
   4. If the new functionality is improved experience, then the Narrative should be "annotated" to describe how the platform improvement would improve that experience. Many times the improvement might not directly impact the Narrative experience and that is ok as long as the improvement is described in the Narrative.
 
 ### Timeline
-  A timeline should include partial, stepwise goals with a rough timeline. For example, what partial functionality or improvements would be available at the end of each sprint.
+  A timeline should include partial, stepwise goals with a rough timeline. For example, what partial functionality or improvements would be available at the end of each sprint. Please avoid excessively detailed implementation plans in this document, though some implementation plans may be needed in order to specify a timeline.
```

Comment:
I would be ok with ballpark estimates; for example, this would take 3 sprints for full functionality